bobtask/map.go

```diff
@@ -3,13 +3,11 @@ package bobtask
 import (
 	"bytes"
 	"fmt"
-	"path/filepath"
-	"sort"
-
-	"github.com/benchkram/bob/pkg/nix"
-
 	"github.com/benchkram/bob/pkg/multilinecmd"
+	"github.com/benchkram/bob/pkg/nix"
 	"github.com/benchkram/errz"
+	"path/filepath"
```

Comment:
Was this auto formatted by your IDE?

Reply:
Yes, my IDE used gofmt import ordering. I switched to use goimports now